inc/broadcast/namespace.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace Altis\Analytics\Broadcast;
+
+use WP_Block_Editor_Context;
+use const Altis\Accelerate\PLUGIN_FILE;
```

Comment:
This puts a hard dependency on this plugin being part of the accelerate plugin. Can we avoid this?

Reply:
Actually we don't use this anymore, so will remove.

Reply:
We haven't really discussed where the menu item will be shown though, the icon is only needed if it'll be a top-level menu. Even then we can adjust the icon location to get rid of this const requirement.

Reply:
What’s your feeling there? Next to global blocks?

Maybe we should have an Accelerate top level item with all features under it. Any thoughts on that @noeltock ?